app/MMB/src/list/Flights.js

```diff
@@ -6,20 +6,21 @@ import { graphql } from 'react-relay';
 
 import type { FlightsQueryResponse } from './__generated__/FlightsQuery.graphql';
 import FlightList from './FlightList';
+import MMBContext from '../MMBContext';
 
-type Props = {|
-  accessToken: string,
+type PropsWithContext = {|
+  accessToken: string | null,
 |};
 
-export default class Flights extends React.Component<Props> {
+export class Flights extends React.Component<PropsWithContext> {
   renderInner = (innerProps: FlightsQueryResponse) => {
     return <FlightList data={innerProps} />;
   };
 
   render = () => (
     <PrivateApiRenderer
       render={this.renderInner}
-      accessToken={this.props.accessToken}
+      accessToken={this.props.accessToken || ''}
```

Comment:
It should work a little bit differently. You didn't maybe notice but we have 2x `PrivateApiRenderer` component. You are using the one directly from `mobile-relay` but you should never do that. There is also other one in `app/core/src/components/relay/PrivateApiRenderer.js` - this component should be consuming the context (probably not MMB context). This way you don't have to deal with access tokens in your code at all (look at how are you prop-drilling the `accessToken` property here just because of `PrivateApiRenderer` component). 🙂

Changes are probably going to be necessary but the idea is clear - do not work with access tokens in your components. I can explain in person if you want...